_pages/laptop-setup.md

```diff
@@ -49,6 +47,16 @@ While you are welcome to customize your laptop, here are some tools that have wo
 [Virtualenvwrapper]: http://virtualenvwrapper.readthedocs.org/en/latest/#
 [Zsh]: http://www.zsh.org/
 
+## Licensed Software
+
+* [Docker] for all your containerization needs
+
+  The docker daemon and docker CLI tools are available on a free license.
+
+  [Docker Desktop] requires a paid license. Ask in #admins-docker in Slack to get a license.
```

Comment:
An easy way to link the Slack channel is replacing the channel with `[#admins-docker](https://gsa-tts.slack.com/app_redirect?channel=admins-docker)` .

Reply:
Thank you, Sarah, added!